src/lib/api.js

```diff
@@ -2,6 +2,43 @@ const request = require('request-promise-native');
 
 const archivingApi = require('./archiving-api');
 const environment = require('./environment');
+const log = require('../lib/log');
+
+const logDeprecatedTransitions = (settings) => {
+  const uri = `${environment.instanceUrl}/api/v1/settings/deprecated-transitions`;
+
+  return request({ uri, method: 'GET', json: true})
+    .then(transitions => {
+      const appSettings = JSON.parse(settings);
+
+      if (!appSettings.transitions) {
```

Comment:
Consider moving this before making the request and skip making the request altogether if there are no transitions to check!.

Reply:
Oooh that's right! Thanks! This is fixed now.